index.js

```diff
@@ -64,6 +64,12 @@ if (require.main === module) {
         // eslint-disable-next-line no-console
         console.warn('Duplicate: %s', duplicate);
       });
+      if (
+        duplicates.length &&
+        process.env.npm_lifecycle_event === 'postinstall'
```

Comment:
Is this env variable also set when using yarn?

Reply:
Yes, I checked.

Reply:
Yes, I checked.